tests/std/tests/P0429R9_flat_map/test.cpp

```diff
@@ -431,13 +439,24 @@ void test_insert() {
     assert(it2->first == 10);
     assert(it2->second == 'n');
 
-    const flat_multimap<int, char>::value_type val_pair2{90, 'w'};
-    const auto it3 = fmm.insert(val_pair2);
-    assert(it3->first == 90);
-    assert(it3->second == 'w');
+    flat_multimap<int, char>::iterator it3 = fmm.insert(fmm.cend(), {70, 'p'});
+    assert(it3->first == 70);
+    assert(it3->second == 'p');
 
-    assert(check_key_content(fmm, {10, 10, 90}));
-    assert(check_value_content(fmm, {'n', 'm', 'w'}));
+    const flat_multimap<int, char>::value_type val_pair2{90, 'w'};
+    const auto it4 = fmm.insert(val_pair2);
+    assert(it4->first == 90);
+    assert(it4->second == 'w');
+
+    flat_multimap<int, char>::iterator it5 = fmm.insert(fmm.cend(), std::pair<char, char>{static_cast<char>(70), 'q'});
+    assert(it5->first == 70);
+    assert(it5->second == 'q');
+
+    assert(check_key_content(fmm, {10, 10, 70, 70, 90}));
+    // FIXME: The expected order of the last 3 value element is 'p', 'q', 'w' (no permutation allowed), according to
+    // [associative.reqmts.general]/72. The current implementation gives 'q', 'p', 'w', which is wrong.

```

Comment:
Should say "last 3 value elements" plural. Also, we should cite the WP number when citing paragraphs, because paragraphs are frequently renumbered; this is valid for N4981. (The WP number is optional when citing a stable name by itself.)